src/message.c

```diff
@@ -55,6 +55,9 @@ static int msg_hist_len = 0;
 static FILE *verbose_fd = NULL;
 static int  verbose_did_open = FALSE;
 
+static int  did_warn_clipboard = FALSE;
+static char *warn_clipboard = "W23: Clipboard register not available, using register 0";
```

Comment:
Is an `N_()` needed here so that this message is translatable?

Reply:
Hm, `N_()` or `_()`. ?